Package.swift

```diff
@@ -38,12 +38,14 @@ let package = Package(
     .target(
         name: "SwiftProtobuf",
         exclude: ["CMakeLists.txt"],
+	resources: [.copy("PrivacyInfo.xcprivacy")],
```

Comment:
Oh, looks there might be a tabs vs spacing things going on in this file.

Reply:
Ooops.  Fixed.